### ISSUE TITLE
Fails to display if only doubleSided is used in the material

### ISSUE THREAD

**Issue by cx20** [Reon90/redcube#15]
I tried displaying the new SimpleSkin model in RedCube.js.
However, an error occurs and is not displayed.

[RedCube.js v2.2.0 + SimpleSkin result:](https://rawcdn.githack.com/cx20/gltf-test/46df65d91c2032e1057980d517a17a69b3f5723a/examples/redcube/index.html?category=tutorialModels&model=SimpleSkin&scale=1&type=glTF)

```javascript
redcube.js:8 
Uncaught (in promise) TypeError: Cannot read property 'baseColorFactor' of undefined
```

Probably due to the lack of `pbrMetallicRoughness` in this model's `material`, but this is _not_ a required item.
```json
  "materials" : [ {
    "doubleSided" : true
  } ],
```

**Comment by Reon90** [comment]
@cx20 was fixed in 2.3.0

**Comment by cx20** [comment]
Thanks! I have confirmed that the latest version resolves the issue.

[RedCube.js v2.3.0 + SimpleSkin result:](https://rawcdn.githack.com/cx20/gltf-test/bbe954fbb47ee78ec5f0233e2dd340b5016a973e/examples/redcube/index.html?category=tutorialModels&model=SimpleSkin&scale=1&type=glTF)
